lib/ansible/plugins/filter/to_json.yml

```
@@ -42,8 +43,11 @@ DOCUMENTATION:
     separators:
       description: The C(item) and C(key) separator to be used in the serialized output,
                    default may change depending on O(indent) and Python version.
-      default: "(', ', ': ')"
-      type: tuple
+      default:
+        - ','
```

Don't forget the space:

```suggestion
        - ', '
```

---

the space is not used in the code, was 'bug' of original docs

---

Even though it's only used for docs, it should still be correct.

---

Just to be clear, the default for `separators` for `json.dumps` is: `(', ', ': ')` [1]

That default is used for the `to_json` filter, since we don't pass that arg by default:

https://github.com/ansible/ansible/blob/6198c7377f545207218fe8eb2e3cfa9673ff8f5e/lib/ansible/plugins/filter/core.py#L69-L83

Which can be seen when using the filter:

```yaml
    - debug:
        msg: "{{ [1, 2] | to_json }}"
```

Notice the space after the comma in the output:

```
ok: [localhost] => {
    "msg": "[1, 2]"
}
```

[1] https://docs.python.org/3/library/json.html#basic-usage

---

he, in the function we explicitly pass `: ` with space and `,` w/o it ... not confusing at all!